dev/tests/js/jasmine/tests/app/code/Magento/Ui/frontend/js/model/messages.test.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'uiRegistry',
+    'Magento_Ui/js/model/messages'
+], function (registry, Constr) {
+    'use strict';
+
+    describe('Magento_Ui/js/model/messages', function () {
+        var obj = new Constr({
```

Comment:
A fresh object should be created `beforeEach` otherwise tests won't be isolated and they should be.